apps/ledger-live-mobile/src/components/Web3AppWebview/NetworkError.tsx

```diff
@@ -20,7 +20,7 @@ export const NetworkError = ({ handleTryAgain }: { handleTryAgain: () => void })
   const { locale } = useLocale();
 
   return (
-    <Flex flex="auto">
```

Comment:
Could we put back what was present before too please ?
Or is it causing some issue ?

`flex={1} justifyContent="center"`